web/docs/decoupled-kit-reposistory-overview.md

```diff
@@ -0,0 +1,108 @@
+#### `cms-kit`
+
+The [`cms-kit`](https://www.npmjs.com/package/@pantheon-systems/cms-kit) exports utilities used in both [`drupal-kit`](#drupal-kit) and [`wordpress-kit`](#wordpress-kit). Those utilities are re-exported from the CMS specific packages for use in the starters.
+
+#### `drupal-kit`
+The [`drupal-kit`](https://www.npmjs.com/package/@pantheon-systems/drupal-kit) utilizes [`@gdwc/drupal-state`](https://www.drupal.org/project/drupal_state), a TypeScript library for fetching and storing data from Drupal's JSON:API.  `@gdwc/drupal-state` lives on https://drupal.org, and is maintained by the Decoupled Kit team along with members of the Drupal community. Please visit https://www.drupal.org/project/drupal_state for more information on DrupalState.
+
+`drupal-kit` exports a `PantheonDrupalState` class that utilizes some methods from [`cms-kit`](#cms-kit) to efficiently fetch and cache data from Drupal on Pantheon.
+
+#### `wordpress-kit`
+
+The [`wordpress-kit`](https://www.npmjs.com/package/@pantheon-systems/wordpress-kit) exports a light wrapper around `graphql-request` which is used to make GraphQL requests to a WordPress instances using the [WPGraphQL](https://wordpress.org/plugins/wp-graphql/) plugin.
+
+It also exports a [`tailwindcss`](https://tailwindcss.com/) plugin which maps WordPress Block Editor styles to tailwindcss classes. Import the plugin into your tailwindcss config to see the Block Editor styles on your content sourced from WordPress.
+
+
+#### `nextjs-kit`
+
+The [`nextjs-kit`](https://www.npmjs.com/package/@pantheon-systems/nextjs-kit) exports shared components used in the Next.js based starter kits, as well as some shared helpers.
+
+## Backend
+
+To facilitate the Frontend Starter Kits, we also maintain WordPress and Drupal projects as starting points for the CMS side of things. These templates are open source and include community and Decoupled Kit managed WordPress plugins and Drupal modules.
+
+Read on for a brief overview of each project. Please see the `README.md` in each project's repo for more detailed information.
+
+### WordPress
+#### Decoupled WordPress Composer Managed
+
+[Decoupled WordPress Composer Managed](https://github.com/pantheon-upstreams/decoupled-wordpress-composer-managed) is a WordPress template designed to get you started with a headless WordPress instance. It includes a number of both community and Decoupled Kit maintained plugins including  [WPGraphQL](https://wordpress.org/plugins/wp-graphql/), [WPGatsby](https://wordpress.org/plugins/wp-gatsby/), and [Pantheon Decoupled](#WordPress Pantheon Decoupled).
+
+#### WordPress Pantheon Decoupled
+
+The [WordPress Pantheon Decoupled](https://github.com/pantheon-systems/wp-pantheon-decoupled) helps to enable headless WordPress without any extra configuration needed by the user. The required plugins are installed and some sample content is included, like a post with an image and an example menu.
+
+#### WordPress Pantheon Decoupled Gatsby
```

Comment:
I'd remove this for now. Since all this plugin does at the moment is add Gatsby as a dependency we're currently adding the wp-gatsby dependency to the composer project directly.